app/views/articles/index.html.erb

```diff
@@ -23,7 +23,11 @@
   <meta name="twitter:card" content="summary_large_image">
   <%= auto_discovery_link_tag(:rss, "https://dev.to/feed", title: "The Practical Dev RSS Feed") %>
 <% end %>
-
+<% if @home_page %>
+  <style>
+    body { background: #f0eaff !important; }
```

Comment:
But but but `--theme-background: #f0eaff;` 😜 Maybe next year

Reply:
To be precise you'd have needed to attach it to the root and it would have only worked with browsers new enough (no IE).

```css
:root { --theme-background: #f0eaff; }
```